src/test/java/uk/gov/hmcts/darts/cases/mapper/HearingEntityToCaseHearingTest.java

```diff
@@ -93,6 +94,33 @@ void testMappingToHearingsWithLegacyAutomatedTranscripts() {
         assertEquals(1, hearingList.get(0).getTranscriptCount());
     }
 
+    @Test
+    void testMappingToHearingsWithTranscriptsWithHiddenDocument() {
+        List<HearingEntity> hearings = CommonTestDataUtil.createHearings(5);
+        var hearingTranscripts = hearings.get(0).getTranscriptions();
+        var transcriptDocs = hearingTranscripts.get(0).getTranscriptionDocumentEntities();
+        transcriptDocs.get(0).setHidden(true);
+
+        List<Hearing> hearingList = HearingEntityToCaseHearing.mapToHearingList(hearings);
+
+        assertEquals(0, hearingList.get(0).getTranscriptCount());
+    }
+
+    @Test
+    void testMappingToHearingsWithTranscriptsWithOnlyOneHiddenDocument() {
+        List<HearingEntity> hearings = CommonTestDataUtil.createHearings(5);
```

Comment:
Why do we create 5 hearings if we only validate against one of them? could this be set to 1?

Reply:
Reduced to 1, there was no reason 🤷